crates/ide_assists/src/handlers/replace_derive_with_manual_impl.rs

```diff
@@ -47,23 +49,23 @@ pub(crate) fn replace_derive_with_manual_impl(
         return None;
     }
 
-    if !args.syntax().text_range().contains(ctx.offset()) {
+    if !(args.l_paren_token()?.text_range().end() <= ctx.offset()
+        && ctx.offset() <= args.r_paren_token()?.text_range().start())
+    {
```

Comment:
This shouldn't need to change

Reply:
I changed this last time because it suddenly failed to work after some refactoring. After debugging this a while it turns out it was originally guarded by the code following

```rust
args.syntax().token_at_offset(ctx.offset()).find(|t| t.kind() == IDENT)?
```

which gives the `(` token at `ctx.offset()` inside the `args` is an ident, comparatively, `ctx.find_token_at_offset()` will give the `derive` token which will fail the `test_ignore_if_cursor_on_param` test.